torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -640,6 +640,30 @@ graph(%a_quant, %min, %max):
           %r = aten::hardtanh_(%a_quant, %min, %max)
           return (%r) )";
 
+  // aten::elu
+  std::string elu = R"(
```

Comment:
Can we make this more compact? If you define a get_prefix_string_fn(aten_op) you could have a function that does:
``` c++
 std::string generate_op_prefix(std::string aten_op)
        {
           return sprintf("graph(\%a_quant, \%scale,\ %input_scale):
          \%a_dequant = aten::dequantize(\%a_quant)
          \%r = %s(\%a_dequant,  \%scale, \%input_scale\n)",aten_op);
}
```

Reply:
sure, will refactor in future PR